loopout.go

```diff
@@ -136,6 +135,7 @@ func newLoopOutSwap(globalCtx context.Context, cfg *swapConfig,
 			MaxMinerFee:      request.MaxMinerFee,
 			MaxSwapFee:       request.MaxSwapFee,
 		},
+		OutgoingChanSet: request.OutgoingChanSet,
```

Comment:
Do we need to make sure that the there's no duplication in the set?

Reply:
Good point, check added